public/static/locales/en/post.json

```diff
@@ -7,12 +7,20 @@
   "readAction": "Read post",
   "editPost": "Edit Post",
   "editPostAction": "Edit Post",
+  "deletePostAction": "Delete Post",
+  "deletePostSuccess": "Post successfully deleted!",
+  "deletePostError": "There was an error while deleting post.",
   "titleLabel": "Title",
   "languageLabel": "Language",
   "save": "Save",
   "finishAction": "Finish post",
   "noLanguageError": "Please select a language.",
   "emptyTitleError": "Please specify a non-empty title.",
   "draftPublishedSuccess": "Post successfully published!",
-  "postBy": "by"
+  "postBy": "by",
+  "deleteModal": {
+    "title": "Do you want to delete the post?",
```

Comment:
```suggestion
    "title": "Are you sure you want to delete this post?",
```

Reply:
Yes, I know about this, I have fixed it, but didn't pushed it
`
  "deleteModal": {
    "title": "Delete Post",
    "body": "Are you sure you want to delete this post?",
    "delete": "Delete",
    "cancel": "Cancel"
  }
`

Reply:
Awesome, looks good!